dashboard/src/components/QuerySidebar/Schema.tsx

```diff
@@ -76,16 +116,19 @@ export default function Schema({
                       key={col}
                       customStyles={
                         (meta as columnMetadata).isMandatory && !tableName.includes('iasql_')
-                          ? 'pl-8 grid grid-cols-2 gap-2 font-bold'
-                          : 'pl-8 grid grid-cols-2 gap-2'
+                          ? 'pl-8 grid grid-cols-12 gap-1 font-bold'
+                          : 'pl-8 grid grid-cols-12 gap-1'
                       }
                     >
-                      <HBox alignment={align.start}>
+                      <HBox customStyles={'col-span-1'}>
+                        {getIconForDataType((meta as columnMetadata).dataType)}
+                      </HBox>
+                      <HBox alignment={align.start} customStyles={'col-span-5'}>
                         <p className='text-ellipsis overflow-hidden' title={col}>
                           {col}
                         </p>
                       </HBox>
-                      <HBox alignment={align.start}>
+                      <HBox alignment={align.start} customStyles={'col-span-6'}>
```

Comment:
I'd drop this entire `<HBox>` entirely and just turn it into a tooltip on the icon.

Reply:
<img width="349" alt="image" src="https://user-images.githubusercontent.com/5357435/230665614-403915fb-3c48-4ceb-93f0-6f35b1842625.png">
<img width="314" alt="image" src="https://user-images.githubusercontent.com/5357435/230665654-a234f09a-f1fa-4645-85d0-65c928bf07e3.png">

Reply:
hover:
<img width="175" alt="image" src="https://user-images.githubusercontent.com/5357435/230665724-f830aca1-f66a-47b2-8b3c-308013649030.png">